test_runner/src/main/kotlin/ftl/ios/xctest/common/ParseObjTests.kt

```diff
@@ -0,0 +1,27 @@
+package ftl.ios.xctest.common
+
+import ftl.config.FtlConstants
+import ftl.util.Bash
+
+internal fun parseObjcTests(binary: String): List<String> {
+    installBinaries
+    validateIsFile(binary)
+
+    val results = mutableListOf<String>()
+    // https://github.com/linkedin/bluepill/blob/37e7efa42472222b81adaa0e88f2bd82aa289b44/Source/Shared/BPXCTestFile.m#L18
+    // must quote binary path in case there are spaces
+    var cmd = "nm -U ${binary.quote()}"
```

Comment:
i am not sure windows is supported?

Reply:
Me too, this code is untouched, only moved from:
https://github.com/Flank/flank/blob/ab55fbcb02f6cfe99b3b502db6bf52f661cd34ca/test_runner/src/main/kotlin/ftl/ios/Parse.kt

Everything should work like before. It's in the description: "This pull request SHOULD NOT change the current behavior of the flank."

Reply:
hmmm alright. We may need to test it